templates/user/dashboard/feeds.tmpl

```diff
@@ -63,12 +63,12 @@
 					{{else if eq .GetOpType 7}}
 						<span class="text truncate issue title has-emoji">{{index .GetIssueInfos 1}}</span>
 					{{else if eq .GetOpType 10}}
-						<span class="text truncate issue title has-emoji">{{.GetIssueTitle}}</span>
+						<a href="{{.GetCommentLink}}" class="text truncate issue title has-emoji">{{.GetIssueTitle}}</a>
 						<p class="text light grey has-emoji">{{index .GetIssueInfos 1}}</p>
 					{{else if eq .GetOpType 11}}
 						<p class="text light grey has-emoji">{{index .GetIssueInfos 1}}</p>
 					{{else if (or (or (eq .GetOpType 12) (eq .GetOpType 13)) (or (eq .GetOpType 14) (eq .GetOpType 15)))}}
-						<span class="text truncate issue title has-emoji">{{.GetIssueTitle}}</span>
+						<a href="{{.GetCommentLink}}"  class="text truncate issue title has-emoji">{{.GetIssueTitle}}</a>
```

Comment:
I dislike using `.GetCommentLink` for actions that don't have anything to do with comments. I know that `.GetCommentLink` will return the link to the issue in this case, but it seems sloppy.

IMO, the best solution here is to add `IssueID` and `Issue` fields to the `Action` struct. This would allow us to just use `.Issue.HTMLURL` here, and `.Comment.HTMLURL` in line 66. It would also allow us to get rid of `.GetIssueInfos`, which to me seems like a hack.

Would it be possible to hold off adding these links to a separate PR? The links are unrelated to fixing the deleted-comment bug, and it seems to me that the best way to add support for the links requires some more refactoring.